### ISSUE TITLE
Article Audit: ARIA States & Properties

### ISSUE THREAD

**Issue by scottaohara** [a11yproject/a11yproject.com#670]
While reviewing the ARIA States and Properties articles, I noticed that the linked resource of the different attributes doesn't match the ARIA 1.1 specification, which lists more attributes.  For instance, `aria-modal` and `aria-details` are missing from the linked resource (and the properties article), but are present in the published ARIA 1.1 spec.

These articles should be reviewed to make sure there aren't any other attributes that are missing.

**Comment by RyanAaronGreen** [comment]
I've gone ahead and looked these two articles over! I've added a few of the new properties and updated all links to the documentation. (#702)
